build.gradle

```diff
@@ -258,7 +258,7 @@ configure(javaCodeCheckedProjects) {
     spotbugs {
         excludeFilter = new File(configDir, 'findbugs-exclude.xml')
         sourceSets = [sourceSets.main]
-        toolVersion = '3.1.7'
+        toolVersion = '3.1.12'
```

Comment:
Upgraded the SpotBugs plugin to the latest release. We will need to upgrade to a later release to eliminate the Gradle deprecation warnings coming from this plugin.

Reply:
This upgrade will be addressed in JAVA-3574. I will create another ticket for the future upgrade.